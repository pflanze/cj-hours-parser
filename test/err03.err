Error: can't process file "test/err03.txt"

Caused by:
    0: can't parse line 3, "set foo<> = 2"
    1: can't parse as a processing instruction: "set foo<> = 2"
    2: column 7: expecting whitespace or '=' after the key "foo"
