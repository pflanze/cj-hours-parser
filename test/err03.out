item=ProcessingInstruction { key: Inline("foo"), value: Inline("1") }
    =ProcessingInstruction { key: Inline("foo"), value: Inline("1") }
item=ProcessingInstruction { key: Inline("foo"), value: Inline("2") }
    =ProcessingInstruction { key: Inline("foo"), value: Inline("2") }
